tests/test_sqltrie.py

```diff
@@ -110,7 +110,7 @@ def test_set_get_root(cls):
     assert trie[()] == b"root"
     del trie[()]
     with pytest.raises(KeyError):
-        trie[()]
+        trie[()]  # pylint: disable=pointless-statement
```

Comment:
[C]: Not related. Fixes tests that were broken on CI at the moment.

Reply:
Sorry, that's my fault. I'm still pushing directly to the project, time to do PRs here.